src/Catel.Test/Catel.Test.Shared/Collections/FastObservableCollectionFacts.cs

```diff
@@ -44,6 +50,42 @@ public void ReturnsTrueWhenPendingNotificationsAreListed()
         }
 
         [TestFixture]
+        public class TheNotificationsSuspendedProperty
+        {
+            [TestCase]
+            public void ReturnsFalseAfterDisposing()
+            {
+                var fastCollection = new FastObservableCollection<int>();
+
+                using (fastCollection.SuspendChangeNotifications())
+                {
+                    fastCollection.Add(1);
+
+                    Assert.IsTrue(fastCollection.NotificationsSuspended);
+                }
+
+                Assert.IsFalse(fastCollection.NotificationsSuspended);
+            }
+
+            /// <remarks>
+            /// BUG: This is a misbehavior - will fail if fixed!
```

Comment:
I have one: suspension counter. Which might be the solution too to remove the boolean mentioned above. Give me a try.